src/components/common/AgreementType.vue

```diff
@@ -33,7 +33,7 @@
       </div>
     </div>
 
-    <div v-else-if="isTypeCC">
+    <div v-else-if="isTypeCCC">
```

Comment:
The idea with this getter (and the others) is to abstract away from its value and instead use a human-understandable label. In this case a "CCC" is a common high level term for this type of business.